x/nameservice/internal/types/msgs_test.go

```diff
@@ -0,0 +1,148 @@
+package types
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMsgSetName(t *testing.T) {
+	name := "maTurtle"
+	value := "1"
+	acc := sdk.AccAddress([]byte("me"))
```

Comment:
not clear to me what's going on here. If you instantiate an address using an arbitrary byte string is that used as the private key?

Reply:
this is just creating a dummy account for testing. there is no need fo the priv key here, once we do simulaiton tests we will need to integrate something with priv key i believe

Reply:
i think its ok not to use valid addresses here since the test is focused on the nameservice message